packages/backend/src/api/factory.test.ts

```diff
@@ -26,20 +27,17 @@ export default (QUnit: QUnit) => {
       fakeFetch = sinon.stub(runtime, 'fetch');
       fakeFetch.onCall(0).returns(jsonOk(userJson));
 
-      const payload = await apiClient.users.getUser('user_deadbeef');
+      const response = await apiClient.users.getUser('user_deadbeef');
 
-      if (!payload) {
-        assert.false(true, 'This assertion should never fail. We need to check for payload to make TS happy.');
-        return;
-      }
```

Comment:
`assertResponse` / `assertErrorResponse` helpers were introduced to drop the `!payload` check.